lib/Common/Core/Assertions.h

```diff
@@ -15,9 +15,11 @@ _declspec(selectany) int AssertsToConsole = false;
 
 #if _WIN32
 _declspec(thread, selectany) int IsInAssert = false;
+#elif !defined(__IOS__)
+__declspec(thread, selectany) int IsInAssert = false;
```

Comment:
Yes. It's not supported on iOS. There is no difference between single / double underscore. Single one is old MS style. It could be combined though but left it mostly because of an explicit separation.